src/main/java/com/github/manolo8/darkbot/Main.java

```diff
@@ -113,6 +113,8 @@ public Main(StartupParams params) {
         this.pluginHandler.updatePluginsSync();
         this.pluginHandler.addListener(this);
 
+        if (params.startConfigs() != null) config = configManager.loadConfig(params.startConfigs());
```

Comment:
By doing this here, you're loading the config twice, things like the API are comming from the main config instead of the new one, same for the size of the window.
Make line 66 just `Config config;` 
Initialize the config (calling loadConfig) with either null or the other start config parameter on line 103 (after setupAuth, but before configManager.getAPI)